models/articles.models.js

```diff
@@ -43,3 +43,22 @@ exports.fetchArticles = () => {
       return articles;
     });
 };
+
+exports.updateArticleById = (article_id, inc_votes) => {
+  return db
+    .query(
+      `
+    UPDATE articles
+    SET votes = votes + $1
+    WHERE article_id = $2
+    RETURNING *;
+  `,
+      [inc_votes, article_id]
+    )
+    .then(({ rows }) => {
+      if (rows.length === 0) {
```

Comment:
We need to check whether the article exists or not in order to determined the reason a patch may fail.If it does not exist, it should throw a 404